platform/root/__tests__/index.test.ts

```diff
@@ -98,3 +98,23 @@ test('calls onShutdown param on "shutdown"', async () => {
   expect(onShutdown).toHaveBeenCalledTimes(1);
   expect(onShutdown).toHaveBeenLastCalledWith(err);
 });
+
+test('calls shutdown when configuring logger fails', async () => {
+  const logged = jest.fn();
+  console.log = logged;
```

Comment:
__issue:__ I don't really like that we change globals like this, because it usually leads to various side effects here and there, other tests in the same suite will be affected as well. So can we use `spyOn` and clear/reset it in `beforeEach` instead?